lib/vsc/utils/mail.py

```diff
@@ -72,10 +72,10 @@ class VscMail(object):
     def __init__(
         self,
         mail_host='',
-        mail_port=587,
+        mail_port=25,
```

Comment:
weird defaults (port 587 without starttls makes no sense) sure, but why change them? are you using it somewhere where you can't configure it?

Reply:
yes, the scripts that mail in vsc-filesystems-quota only use the mail hostname from vsc-config. I could change it to be tuple in vsc-config `(hostname, port)`. But that is a bit more work and needs changes on multiple places.

In my opinion, use port 25 as the default makes more sense? I also don't really see good reason to use 587 as long as you use starttls?

Reply:
well, port 25 is not supposed to be used for client-mailserver, only mailserver-mailserver. 
it's best if we can somehow configue it properly (even enabling/disabling starttls)

Reply:
@wpoely86 perhaps allow the smtp option to indicate a filename with the mailconfig  if it starts with a `/`?

Reply:
Actually upon better reading the code, we could just provide `server:25` and that will also work.

But I would suggest that we flip on starttls by default?

Reply:
Alternatively we can add a default configuration file and gracefully handle it if it's not present?

Reply:
Is 25 only server-server? Didn't know that. I thought the only difference was that 587 enforces TLS.